hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/HoodieSimpleBucketIndex.java

```diff
@@ -61,6 +63,15 @@ public Map<Integer, HoodieRecordLocation> loadBucketIdToFileIdMappingForPartitio
           if (!bucketIdToFileIdMapping.containsKey(bucketId)) {
             bucketIdToFileIdMapping.put(bucketId, new HoodieRecordLocation(commitTime, fileId));
           } else {
+            // If hoodie.write.bucketid.conflict.rollback.enable enable, rollback unfinished replacement instant
```

Comment:
1. Rollback the `replacement` commit is not a good solution. Because it the not the only one possibility caused multiple base fileids belongs to the same bucket
2. Besides, rollback in the foreach loop does not make sense.

Reply:
@xuzifu666 Could we introduce a CallProcedure to do check and clean operation?
And left a message to advice the user to trigger this CallProcedure?

Reply:
> @xuzifu666 Could we introduce a CallProcedure to do check and clean operation? And left a message to advice the user to trigger this CallProcedure?

Good Idea，this is a better way. @beyond1920

Reply:
Had add a procedure to support it and left a message to advice user to call it. @beyond1920